include/aspect/postprocess/sealevel.h

```diff
@@ -0,0 +1,150 @@
+/*
+  Copyright (C) 2011 - 2019 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_postprocess_sealevel_h
+#define _aspect_postprocess_sealevel_h
+
+#include <aspect/postprocess/interface.h>
+#include <aspect/simulator_access.h>
+
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+
+    /**
+     * A postprocessor that outputs the sea level to file.
+     *
+     * @ingroup Postprocessing
+     */
+    template <int dim>
+    class SeaLevel : public Interface<dim>, public ::aspect::SimulatorAccess<dim>
+    {
+      public:
+        /**
+         * Sets up structured data lookup for topography and ice height input data.
+         */
+        void initialize() override;
+
+        /**
+         * Output non-uniform sea level change [m] to file.
```

Comment:
As far as I understood there is a uniform and non-uniform part to the sea level output. Will this function only write the non-uniform part or the sum between uniform and non-uniform part? Update the comment accordingly.